lib/components/user/monitored-trip/saved-trip-list.js

```diff
@@ -10,25 +10,14 @@ import * as userActions from '../../../actions/user'
 import BackToTripPlanner from '../back-to-trip-planner'
 import AwaitingScreen from '../awaiting-screen'
 import TripSummaryPane from './trip-summary-pane'
-import { CREATE_ACCOUNT_PATH, TRIPS_PATH } from '../../../util/constants'
+import { TRIPS_PATH } from '../../../util/constants'
 import { RETURN_TO_CURRENT_ROUTE } from '../../../util/ui'
 import withLoggedInUserSupport from '../with-logged-in-user-support'
 
 /**
  * This component displays the list of saved trips for the logged-in user.
  */
 class SavedTripList extends Component {
-  componentDidMount () {
-    const { loggedInUser, routeTo } = this.props
-
-    if (!loggedInUser.hasConsentedToTerms) {
```

Comment:
This check should remain because this component can be directly accessed from the URL `/account/trips` even if the user did not finish setting up their account.

Reply:
Ok, please disregard.